lib/sidekiq/statistic/helpers/color.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Sidekiq
+  module Statistic
+    module Helpers
+      class Color
+        def self.color_for(worker)
```

Comment:
What do you think to rename the method to `for` to avoid being redundant? We could call it just `Color.for(...)`
We could also rename the parameter itself to be more clear, `worker` seems to be just a `term` or `name`, not a worker itself.